### PR TITLE
Here be Dragons

### PR DESCRIPTION
**Pull request by jaanonim**

### CODE REVIEW

**Comment by jaanonim** [comment]
I added 2 lines of code that nobody has added to since the last 2 months.

Seriously, I am fixing this stupid zoom button.

File: src/components/MainMap.vue [hunk 1]
```diff
@@ -4,7 +4,12 @@
       @click="setSidebar"
       :zoom="zoom"
       :center="center"
-      :options="{zoomControl: false}"
+      :options="{
+        zoomControl: false,
+        zoomControlOptions: {
+          position: 6
+        }
+      }"
```
Comment (Wuzado) [inline src/components/MainMap.vue:12]: It already is implemented though in the commit eb85a03? On line 14: 
`<l-control-zoom position="bottomright"></l-control-zoom>`

Its position is set in a clearer HTML-style attribute.

File: src/components/DisabilitiesDesc.vue [hunk 2]
```diff
@@ -10,9 +10,9 @@
           aria-controls="contentIdForA11y3">
           <a class="card-header-icon">
             <b-icon style="color:green;" v-if="levelOfSupport === 2" class="fas fa-thumbs-up"></b-icon>
-            <b-icon style="color:gold;" v-if="levelOfSupport === 1" class="fas fa-exclamation-triangle"></b-icon>
-            <b-icon style="color:grey;" v-if="levelOfSupport === 0" class="fas fa-question-circle"></b-icon>
-            <b-icon style="color:red;" v-if="levelOfSupport === -1" class="fas fa-thumbs-down"></b-icon>
+            <b-icon style="color:gold;" v-else-if="levelOfSupport === 1" class="fas fa-exclamation-triangle"></b-icon>
+            <b-icon style="color:grey;" v-else-if="levelOfSupport === 0" class="fas fa-question-circle"></b-icon>
+            <b-icon style="color:red;" v-else-if="levelOfSupport === -1" class="fas fa-thumbs-down"></b-icon>
```
Comment (Wuzado) [inline src/components/DisabilitiesDesc.vue:15]: This code change is actually quite valid, but here, logically, it shouldn't ever, ever have two values at once.

File: src/store/store.js [hunk 3]
```diff
@@ -73,7 +73,7 @@ export const store = new Vuex.Store({
     async getCurrentPoint(context, id) {
       let currentID = id;
       let response = await instance.get(
-        process.env.VUE_APP_HOSTNAME + "/points/" + currentID,
+        "127.0.0.1:8000" + "/points/" + currentID,
```
Comment (Wuzado) [inline src/store/store.js:76]: Same as line 57.

File: src/store/store.js [hunk 4]
```diff
@@ -54,7 +54,7 @@ export const store = new Vuex.Store({
   actions: {
     async apiInit(context) {
       let response = await instance.get(
-        process.env.VUE_APP_HOSTNAME + "/points/",
+        "127.0.0.1:8000" + "/points/",
```
Comment (Wuzado) [inline src/store/store.js:57]: Why are you setting the hostname like that? You should manually set it as a env var.

(Hint: .env files)

File: src/components/Navbar.vue [hunk 5]
```diff
@@ -3,7 +3,7 @@
     <template slot="brand">
       <b-navbar-item href="/">
         <img
-          src="../../public/about/assets/logo_full.png"
+          src="./../assets/logo_full.png"
```
Comment (Wuzado) [inline src/components/Navbar.vue:6]: Webpack and builder software correctly resolves the path later, during the (both dev and prod) build stage.

File: src/components/MyMarker.vue [hunk 6]
```diff
@@ -9,6 +9,11 @@ export default {
   components: {
     LMarker
   },
+  data(){
+    return{
+      loading: true
+    }
+  },
```
Comment (Wuzado) [inline src/components/MyMarker.vue:16]: Why is this even here? There isn't a single usage of this anywhere in the project files.

File: src/components/MainMap.vue (hunk 1 above)
Comment (Wuzado) [reply src/components/MainMap.vue:12]: I might have not deployed it correctly last time though. I re-deployed it for now, and I am looking into possibilities of CD on the master branch.

**Review by Wuzado — CHANGES_REQUESTED** [review]
This PR doesn't introduce many real changes, but gives some food for thought and ideas how to improve the site.

**Comment by Wuzado** [comment]
This PR will be closed after :
* [ ] introducing CD
* [ ] changing v-if's to v-else-if's

Also, please, don't use whatthecommit.com for your commit messages. (Yes, I know about squashing)

**Pull request closed without merging**